tslearn/utils.py

```diff
@@ -8,6 +8,11 @@
 __author__ = 'Romain Tavenard romain.tavenard[at]univ-rennes2.fr'
 
 
+def _check_dims(X_fit, X):
+    if X_fit.shape[1:] != X.shape[1:]:
+        raise ValueError('Second and third dimension must match!')
```

Comment:
Maybe add info about observed shapes?

Reply:
Good idea, it is more informative. [check_array](https://github.com/scikit-learn/scikit-learn/blob/7813f7efb5b2012412888b69e73d76f2df2b50b6/sklearn/utils/validation.py#L537-L539) and [check_consistent_length](https://github.com/scikit-learn/scikit-learn/blob/7813f7efb5b2012412888b69e73d76f2df2b50b6/sklearn/utils/validation.py#L190-L205) do that for instance.

I think that making the function public and having a docstring would be helpful for future contributors.